th2_common/schema/message/impl/rabbitmq/abstract_rabbit_subscriber.py

```diff
@@ -142,7 +141,7 @@ def handle(self, channel, method, properties, body):
 
         self.handle_with_listener(value, channel, method)
 
-        end_time = time()
+        end_time = time.time()
         process_timer.set(end_time - start_time)
```

Comment:
I think we do not need to create end_time variable. 
We can just put 'time.time()' to 'process_timer.set(end_time - start_time)' instead of 'end_time'.

@RaevskiySV, what do you think about it?

Reply:
Of course we can remove the variable, it's just for code readability, I guess.